Bit-Manipulation/LogTwo.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * https://handwiki.org/wiki/Binary_logarithm
+ * Approximate log2 using only bitwise operators
+ * @param {number} n
+ * @returns {number} Log2 approximation equal to floor(log2(n))
+ */
+export const logTwo = (n) => {
```

Comment:
`bitwiseLogTwo` will be a better name for the function.

Reply:
No. This is still an integer (floor) log two. I wouldn't call `1 << n` "bitwisePowTwo" either; bitwise implies that the logarithm was somehow bitwise, e.g. applied to each bit separately. Implementation details such as it being bitwise don't need to be part of the function name (and it would be redundant with the folder being "Bit-Manipulation").